### PR TITLE
Create confidentiality_agreements.md

### PR DESCRIPTION
**Pull request by EThom73**
What?
Made Tech policy on confidentiality agreements

Why?
ISO 27001 compliance

### CODE REVIEW

File: ims/people/confidentiality_agreements.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ - All employees sign Terms and Conditions of Employment which include clauses for 
+   - Confidentiality
+   - Intellectual property
+ - Company Non-Disclosure Agreements (NDA’s) are included in contracts raised with contractors 
+ - Confidentiality requirements are also written into supplier contracts and agreements
+ - Visitors are not allowed in areas where sensitive information is accessible unless they are working under a current NDA
+   - For more information please see our policy on [Physical Security](company/physical_security.md)
+ - No recording equipment may be used in internal, supplier, or customer meetings without the knowledge and consent of all those present in the meetings
+ 
+ ### Terms & Conditions of Employment
```
Comment (lukemorton) [inline ims/people/confidentiality_agreements.md]: Space before `###` should be removed

Comment (EThom73) [reply ims/people/confidentiality_agreements.md]: 👍 Good spot. Thanks

File: ims/people/confidentiality_agreements.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+This outlines Made Tech's policy in regards to confidentiality agreements.
+
+## The Policy
+
+ - All employees sign Terms and Conditions of Employment which include clauses for 
+   - Confidentiality
+   - Intellectual property
+ - Company Non-Disclosure Agreements (NDA’s) are included in contracts raised with contractors 
+ - Confidentiality requirements are also written into supplier contracts and agreements
+ - Visitors are not allowed in areas where sensitive information is accessible unless they are working under a current NDA
```
Comment (cob16) [inline ims/people/confidentiality_agreements.md]: Is this currently being enforced?

Comment (EThom73) [reply ims/people/confidentiality_agreements.md]: Yep - John from SW has an NDA. Colin our ISO consultant does as well. 
For other visitors (not regularly in office) the rules around physical security and visitors apply.

**Pull request closed without merging**